amoro-shade-thrift/src/main/resources/META-INF/NOTICE

```diff
@@ -1,8 +1,5 @@
 amoro-shade-thrift
-Copyright (C) 2006 - 2019, The Apache Software Foundation
```

Comment:
I think the copyright is right already and that the change is wrong

better still, we should just keep the notice from the actual thrift jar